Week3/homework/1-step3.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 function foo(func) {
-  // What to do here? 
+  return func();
 }
 
 function bar() {
```

Comment:
Excellent.  You are the first of the people I've reviewed so far, to return the result of func() from the function.  Well done.